CMakeLists.txt

```diff
@@ -99,3 +106,24 @@ endif()
 add_subdirectory(matrix_multiplication)
 add_subdirectory(fft)
 add_subdirectory(dependent_index_sets)
+
+set(FIGURES_DIRECTORY "${CMAKE_SOURCE_DIR}/figs")
+
+add_custom_command(
+  OUTPUT rajaMmult.cali
```

Comment:
Unfortunately we'll need to add something like this for each type of caliper file we want.